src/main/java/com/cinema/Main.java

```diff
@@ -129,10 +131,14 @@ public static void main(String[] args) throws AuthenticationException {
         cartService.addSession(matrixRed, roman);
         cartService.addSession(lordOfRingsBlue, pavlo);
         cartService.addSession(lordOfRingsBlue, pavlo);
-        log.info("Roman's shoppingCart :" + cartService.getByUser(roman).toString());
-        log.info("Pavlo's shoppingCart :" + cartService.getByUser(pavlo).toString());
-        ShoppingCart romamSC = cartService.getByUser(roman);
-        cartService.clear(romamSC);
-        log.info("Roman's shoppingCart :" + cartService.getByUser(roman));
+
+        //-----------------------------------------------------------------------------------
+        OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
+        List<Ticket> romanTickets = cartService.getByUser(roman).getTickets();
+        List<Ticket> pavloTickets = cartService.getByUser(pavlo).getTickets();
+        orderService.completeOrder(romanTickets, roman);
+        orderService.completeOrder(pavloTickets, pavlo);
+        orderService.getOrderHistory(roman);
+        System.out.println(cartService.getByUser(roman).toString());
```

Comment:
you can omit explicitly calling `toString()` when you are printing out the object
it is called automatically

Reply:
done